be/src/storage/olap_common.h

```diff
@@ -319,12 +319,18 @@ const char* const kPrefetchHitCount = "prefetch_hit_count";
 const char* const kPrefetchWaitFinishNs = "prefetch_wait_finish_ns";
 const char* const kPrefetchPendingNs = "prefetch_pending_ns";
 
+// The position index of a column in a specific TabletSchema starts from 0.
+// The position of the same column in different TabletSchema may be different, which
+// means that the same column may have a different ColumnId in different contexts, depending
+// on the TabletSchema used.
+// TODO: Change the name
```

Comment:
I totally agree with you, but this is already used by a very large amount of code now, and changing the name would cause a lot of code changes, which I don't intend to do in this patch